knowledge-base/drawer-add-tooltip.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Add tooltips to the Drawer items
+description: How to add tooltips to the Drawer items
+type: how-to
+page_title: Drawer tooltips
+slug: drawer-kb-tooltips
+position:
+tags:
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Drawer for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I would like to add [Tooltips]({%slug tooltip-overview%}) to the [Drawer's]({%slug drawer-overview%}) navigation icons.
+
+
+## Solution
+
+To add a tooltip to the drawer navigation icons you have to use the [ItemTemplate]({%slug drawer-templates%}#itemtemplate) and add a suitable CSS selector to the `TargetSelector` parameter of the Telerik Tooltip.
```

Comment:
Clarify a little - that you should use the item template to add the `title` attribute to the desired element. Then move the info for the telerik tooltip on a separate pargraph and denote it as optional.

Reply:
Done with - 95e8527b6ff641d78b9a65927b27aebf345e4e59